File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -174,139 +162,134 @@ A complete work sample for converting an HTML to PDF in Azure App service on Lin
 
 ### Steps to convert HTML to PDF in Azure app service using the Blink with Linux docker container:
 
-Create a new ASP.NET Core application and enable the docker support with Linux as a target OS.
+1. Create a new ASP.NET Core application and enable the docker support with Linux as a target OS.
 ![Convert HTMLToPDF Azure Docker Step1](htmlconversion_images/DockerStep1.png)
+
+2. Choose your project's target framework, then select Configure for HTTPS and Enable Docker.
 ![Convert HTMLToPDF Azure Docker Step2](htmlconversion_images/DockerStep2.png)
 
-Install the [Syncfusion.HtmlToPdfConverter.Net.Linux](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Linux/) NuGet package as a reference to your .NET Core application [NuGet.org](https://www.nuget.org/)
+3. Install the [Syncfusion.HtmlToPdfConverter.Net.Linux](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Linux/) NuGet package as a reference to your .NET Core application [NuGet.org](https://www.nuget.org/)
 ![Convert HTMLToPDF Docker Step3](htmlconversion_images/AzureDocker1.png)
 
-Include the following commands in the docker file to install the dependent packages in the docker container.
+4. Include the following commands in the docker file to install the dependent packages in the docker container.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-RUN apt-get update && \
+   RUN apt-get update && \
      apt-get install -yq --no-install-recommends \ 
      libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \ 
      libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \ 
      libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 \ 
      libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 \ 
      libnss3 libgbm1
 
-{% endhighlight %}
+   {% endhighlight %}
 
 ![Convert HTMLToPDF Azure Docker Step4](htmlconversion_images/DockerStep4.png)
 
-Add an Export to the PDF button in the index.cshtml.
+5. Add an Export to the PDF button in the index.cshtml.
```

Comment:
Modify like this "Add a new button in the Index.cshtml file."

Reply:
Modified